.github/ISSUE_TEMPLATE/BUG_SDK.md

```diff
@@ -0,0 +1,28 @@
+---
+name: ⚠️ Report SDK Bug
+about: Report a SDK bug on Kubeflow Pipelines
+title: "[sdk]: [Bug Name]"
+labels: kind/bug, area/sdk
+---
+
+### Environment
+
+*  KFP version: <!-- If you are not sure, build commit shows on bottom of KFP UI left sidenav. -->
```

Comment:
nit: this is outdated, now KFP UI shows exact version numbers in the bottom of sidenav.

Reply:
Thank you! Updated.